pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -316,15 +335,32 @@ func (a *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) error
 			}
 			// update the message
 			acs.Message = fmt.Sprintf("The workload got reservation on %q", reservingRemote)
+			// update the transition time
```

Comment:
The comment explains perfectly readable line, but it is more important to write **why**, because it often depends on interaction with other parts of the system, which is not clear by looking at the lines below. Tbh, I don't know why it is updated.

Reply:
done